src/panels/ExplorerPanel.tsx

```diff
@@ -0,0 +1,147 @@
+import { FileEntry } from '@tauri-apps/api/fs';
+import { useAtomValue, useSetAtom } from 'jotai';
+import { useEffect, useState } from 'react';
+import { VscFile, VscFolder, VscSplitHorizontal } from 'react-icons/vsc';
+
+import { leftPaneAtom, openFileInPaneAtom, rightPaneAtom, splitFileToPaneAtom } from '../atoms/openFilesState';
+import { PanelSection } from '../components/PanelSection';
+import { PanelWrapper } from '../components/PanelWrapper';
+import { cx } from '../cx';
+import { readAllProjectFiles } from '../filesystem';
+
+export function ExplorerPanel() {
+  const left = useAtomValue(leftPaneAtom);
+  const right = useAtomValue(rightPaneAtom);
+  const openFileInPane = useSetAtom(openFileInPaneAtom);
+  const splitFileToPane = useSetAtom(splitFileToPaneAtom);
+
+  const [allFiles, setFiles] = useState<FileEntry[]>([]);
+  useEffect(() => {
+    (async function refreshProjectTree() {
+      const newFiles = await readAllProjectFiles();
+      setFiles(newFiles);
+    })();
+  }, [setFiles]);
+
+  return (
+    <PanelWrapper title="Explorer">
+      <PanelSection title="Open Files">
+        {right.files.length > 0 && <div className="ml-4 text-xs font-bold">LEFT</div>}
```

Comment:
I think it should be `left.files.` here

Reply:
It shouldn't. I will make this rule more clear with a variable. 
We only want to display group names if the right panel has files.